.github/workflows/test-docs.yml

```diff
@@ -0,0 +1,14 @@
+name: "Test docs build"
+on: 
+  pull_request:
+    branches: [master]
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: |
+        cd docs/
+        pip install -r requirements.txt
+        sphinx-build -M html . _build -W
```

Comment:
This'll definitely work. For slightly nicer formatting when viewing the build logs I'd maybe suggest splitting up the steps and naming them:

```
    - name: clone sotodlib
      uses: actions/checkout@v2

    - name: install dependencies
      working-directory: ./docs
      run: pip3 install -r requirements.txt

    - name: build docs
      working-directory: ./docs
      run: sphinx-build -M html . _build -W
```

Reply:
Ah, working-directory ... I like it ...